docs/src/modeling/forms/links/README.md

```diff
@@ -0,0 +1,37 @@
+# Verlinkung von Webseiten oder Dokumenten an Nutzeraufgabe 
+
+DigiWF stellt die Möglichkeit bereit als Prozessmodellierer dem Sachbearbeiter zusätzliche Informationen bereitzustellen.
+
+Die Links können dazu über eine Prozessvariable erstellt werden.
+
+Die Links werden unter dem Taskname angezeigt.
```

Comment:
"in der Detailansich eines Tasks" oder so

Screenshot wäre auch gut

Reply:
behoben